src/equations/equations_parabolic.jl

```diff
@@ -2,11 +2,17 @@
 # specialize this function to compute gradients e.g., of primitive variables instead of conservative
 gradient_variable_transformation(::AbstractEquationsParabolic) = cons2cons
 
+# By default, the gradients are taken with respect to the conservative variables.
+# this is reflected by the type parameter `GradientVariablesConservative` in the abstract
+# type `AbstractEquationsParabolic{NDIMS, NVARS, GradientVariablesConservative}`.
+abstract type AbstractGradientVariable end
+struct GradientVariablesConservative <: AbstractGradientVariable end
```

Comment:
If this is defined here, shouldn't we also move the `GradientVariablesPrimitive` and `GradientVariablesEntropy` here from NSE 2D? They are not very well-placed there anyways, and having them all at the same place (I expect that this list of potential gradient variables is probably complete, or at least it will not grow indefinitely) would make it easier to reason about the options one potentially has.

Reply:
I think that makes sense. I'll move them in the morning